seed/ts-sdk/nullable/src/api/resources/nullable/client/Client.ts

```diff
@@ -76,6 +77,10 @@ export class Nullable {
             }
         }
 
+        if (extra !== undefined) {
+            _queryParams["extra"] = extra?.toString ?? null;
+        }
```

Comment:
I thought it was a typo in the description, but I think `.toString` should be invoked.

Reply:
Oof, surprised this works at all then - taking a closer look.